etc/profile-m-z/patch.profile

```diff
@@ -43,7 +43,7 @@ x11 none
 
 private-bin patch,red
 private-dev
-private-lib libfakeroot
+private-lib libfakeroot libdl.so.*
```

Comment:
> Would it make sense
for it to be part of sort.py?

No, sort.py is for sorting. This would be out of scope and a own script would be better.

> What do you think about failing/warning on these cases?

How do you detect these causes? `^private-lib .* lib.*` (space before "lib")?

Reply:
> > Would it make sense
> > for it to be part of sort.py?
> 
> No, sort.py is for sorting. This would be out of scope and a own script would
> be better.

I see, and I do like "do one thing and do it well".  I'll add something to my
WIP linter branch then.
 
> > What do you think about failing/warning on these cases?
> 
> How do you detect these causes? `^private-lib .* lib.*` (space before "lib")?

With the regex that I used on my last comment:

```sh
grep -nr '^[^ ]*private-lib [0-9A-Za-z*,.-]* ' etc
```

It just has to detect any space after any given path.

I wanted to also catch commented-out entries (e.g.: `#private-lib`), so that's
why it has `^[^ ]*` (though it might have been better as `^#*` instead).  But
it can be simplified:

```sh
grep -nr '^private-lib [0-9A-Za-z*,.-]* ' etc
```

Even further:

```sh
grep -nr '^private-lib [^ ]* ' etc
```

Example:

```console
$ git show --pretty='%h %ai %s' -s
42e2db127 2021-02-20 10:06:58 -0500 jaitest - simple sandbox testing utility program
$ grep -nr '^private-lib [^ ]* ' etc
$
$ git show --pretty='%h %ai %s' -s
3a68420fe 2021-02-20 18:59:18 +0100 Fix patch-util not having access to libdl.so
$ grep -nr '^private-lib [^ ]* ' etc
etc/profile-m-z/patch.profile:46:private-lib libfakeroot libdl.so.*
```

It also matches `private-lib  ` (double spaces), but I see that as a bonus
linter feature :p